### ISSUE TITLE
Accessibility Concerns

### ISSUE THREAD

**Issue by peschkaj** [legitbiz/SealsOfFate#125]
Currently, we've taken no aspect of accessibility into account. We should examine accessibility tooling and see how we can use it to assist our development process.

One issue that I am immediately aware of is our green on red health bar.

**Comment by TsebMagi** [comment]
There's an android app that will filter colors into the different
colorblind spectrums we can use to investigate our UI.

**Comment by peschkaj** [comment]
Yeah, I have one of those android apps because it's fun to look at crap with it. 

I specifically created this issue as a placeholder so we make sure accessibility is a feature that we include, rather than adding it at the end, where it becomes more expensive and time consuming.

**Comment by connorkuehl** [comment]
Can we configure some sort of "Command" pattern with UI color elements?
Basically mapping to a color based on if a "color blindness" bool is
flipped on?

See "Command" chapter in Game Programming Patterns:

http://gameprogrammingpatterns.com/command.html
